test/radiator_web/graphql/admin/schema/query/episodes_test.exs

```diff
@@ -97,6 +97,51 @@ defmodule RadiatorWeb.GraphQL.Admin.Schema.Query.EpisodesTest do
            }
   end
 
+  @single_guid_query """
+  query ($guid: String!) {
+    episode(guid: $guid) {
+      id
+      title
+      guid
+    }
+  }
+  """
+
+  test "episode by guid returns an episode", %{conn: conn, user: user} do
+    episode = insert(:published_episode, title: "Foo", guid: "guid-foo") |> owned_by(user)
+
+    conn = get conn, "/api/graphql", query: @single_guid_query, variables: %{"guid" => "guid-foo"}
+
+    assert json_response(conn, 200) == %{
+             "data" => %{
+               "episode" => %{
+                 "id" => Integer.to_string(episode.id),
+                 "title" => episode.title,
+                 "guid" => "guid-foo"
+               }
+             }
+           }
+  end
+
+  test "episode by guid returns an episode when non existent", %{conn: conn} do
```

Comment:
shouldn't the test names be positive? E.g. `returns not found when non-existent` (and apparently, non-existent is hyphenated according to the macOS dictionary)